src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -23,9 +23,16 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
+import api.support.fixtures.ItemsFixture;
```

Comment:
Please can these imports be ordered according to https://issues.folio.org/browse/FOLIO-1667

I can push an update for this if needed?